corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -182,6 +182,16 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
               {% input_trans module.search_config.title_label langs input_name='title_label' input_id='title_label' data_bind="value: title_label" %}
             </div>
           </div>
+          <div class="form-group">
+            <label for="search_screen_subtitle" class="{% css_label_class %} control-label">
```

Comment:
The value for the `for` attribute should be the id of the input, so currently `description_text`